services/core-web/src/components/modalContent/ViewNoticeOfDepartureModal.js

```diff
@@ -55,12 +55,8 @@ const propTypes = {
 };
 
 export const renderContacts = (props) => {
-  const { fields, formValues } = props;
-  if (fields.length > 0 && formValues.nod_type === NOD_TYPE_FIELD_VALUE.NON_SUBSTANTIAL) {
-    fields.pop();
-  } else if (fields.length < 1 && formValues.nod_type !== NOD_TYPE_FIELD_VALUE.NON_SUBSTANTIAL) {
-    fields.push({ is_primary: true });
-  }
+  const { fields } = props;
+
```

Comment:
This is the only chunk where we were using the `formValues` prop, so we can remove it from the rest of this page.
That would be:
`MapStateToProps`
`propTypes`
the destructured props
`FieldArray`